essd/mat-map.R

```diff
@@ -0,0 +1,59 @@
+# Script to extract terrestrial MAT and MAP data from WorldClim
+# Created April 2020 | Stephanie Pennington
+
+# Load libraries
+library(raster)
+library(sp)
+library(dplyr)
+library(cosore)
+library(tidyr)
+library(ggplot2)
+library(RColorBrewer)
+
+# Download worldclim data for precip and tmean, store in essd/ folder
+precip <- getData("worldclim", path = "essd/", var="prec", res = 10)
```

Comment:
If the WorldClim data are in `essd/`, and this script is there too, should the `path` parameter just be "./"?

Reply:
This function puts the WorldClim data in `essd/`. Since the working directory is just `cosore/`, the data goes there by default.

Reply:
@bpbond Everything is up to date on my end. I addressed your comment abt the path, see above. Also, lines 14 and 15 download the WorldClim data, so you shouldn't need my local files right?